src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] namesOfArticles = {"supply", "buy", "result"};
+        int[] values = readStatistic(fromFileName, namesOfArticles);
+        writeStatistic(toFileName, namesOfArticles, values);
```

Comment:
<img width="766" alt="image" src="https://github.com/mate-academy/jv-work-with-file/assets/81781479/03d7a285-9bd6-4f22-b0bd-bda7590412d1">

reading and parsing should be in separate methods